samples/3d-platformer/build.gradle.kts

```diff
@@ -4,11 +4,11 @@ plugins {
 }
 
 repositories {
-    mavenLocal()
     jcenter()
 }
 
 
 godot {
     debug.set(true)
+    defaultPlatforms()
```

Comment:
I also suggest we change the name of our `Platform` enum to `GodotPlatform` as the IDE has trouble importing the correct one and you always have to import the correct ones yourself

Reply:
Let's do that in a separate PR, not here.

Reply:
:+1: